techniques/failures/F46.html

```diff
@@ -70,8 +70,9 @@
                                     table element</li>
             <li>
                <p>If the table is used only to visually lay out elements within the
```

Comment:
We've done the setup line in a different way usually - e.g., "For each table used only to visually lay out elements within the page:"